### ISSUE TITLE
Windows -T %T issue

### ISSUE THREAD

**Issue by Beercow** [simsong/tcpflow#138]
On Windows, using the -T %T option causes the following error

tcpflow.exe: Cannot open: 2014-06-06T00:47:56Z

This is because you cannot have a : in a windows file name.

Is there any workaround for this?

**Comment by simsong** [comment]
Yes, you can specify your own template. How would you like this template to
work on Windows machines?
